### PR TITLE
Update docker-compose.yml

### LINKED ISSUE #4091: Write `docker-compose.yml` file that spins up our backend with the web frontend using neutrino
This is useful to give devs a oneliner for spinning up a working enivironment locally. We can use neutrino as the backend so that there is zero configuration required from the developer (no connecting to bitcoind). The default network should probably be testnet?

### PR DESCRIPTION
**Pull request by rorp**
Fixes https://github.com/bitcoin-s/bitcoin-s/issues/4091

### CODE REVIEW

**Comment by Christewart** [comment]
I get this error on startup

Screenshot from 2022-03-24 13-29-58

File: docker-compose.yml [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
-      - ./website/pages:/app/website/pages
-      - ./website/static:/app/website/static
-      - ./website/sidebars.json:/app/website/sidebars.json
-      - ./website/siteConfig.js:/app/website/siteConfig.js
-    working_dir: /app/website
+      - ./data/wallet:/home/bitcoin-s/.bitcoin-s
+    environment:
+      BITCOIN_S_NODE_MODE: neutrino
+      BITCOIN_S_PROXY_ENABLED: "false"
+      BITCOIN_S_TOR_ENABLED: "false"
```
Comment (Christewart) [inline docker-compose.yml:34]: I think we want this enabled no? Otherwise they won't be able to negotiate a DLC?

**Comment by rorp** [comment]
Does your `docker-compose` process have permissions to write into `./data` directory?

**Comment by Christewart** [comment]
> Does your `docker-compose` process have permissions to write into `./data` directory?

Is this the right command to run? Let me know if you want me to run something else

Screenshot from 2022-03-24 13-55-40

**Comment by Christewart** [comment]
What i get when running on my m1 mac

Screen Shot 2022-03-25 at 7 42 38 AM

File: docker-compose.yml [hunk 2]
```diff
@@ -1,17 +1,38 @@
-version: "3"
+version: "3.7"
 
 services:
-  docusaurus:
-    build: .
+  web:
+    image: bitcoinscala/wallet-server-ui:1.9.0-9ca96ac1-SNAPSHOT
+    user: "1000:1000"
```
Comment (Christewart) [inline docker-compose.yml]: If i change this to `root` and below in the `walletserver` service to `root` this error no longer occurs on my linux machine.: 

https://github.com/bitcoin-s/bitcoin-s/pull/4218#issuecomment-1077923564

I'm following this answer on SO: https://stackoverflow.com/questions/48727548/how-to-configure-docker-compose-yml-to-up-a-container-as-root

**Pull request merged**